### DISCUSSION TITLE
这个游戏引擎的目的是啥？

### DISCUSSION THREAD

**Discussion by lizyChy0329** [galacean/discussion#131] [Engine] [answered]
目前看是专供阿里内部做营销游戏使用的
1. 有收费计划吗？
2. 有 roadmap 吗？
3. 我可以用这来做移动端网页吗？

**Comment by gz65555** [comment, marked as answer]
1. 暂时没有
2. wiki 里有 roadmap
3. 可以在移动端渲染可莉
